CHANGELOG.rst

```diff
@@ -0,0 +1,49 @@
+OMEGO version history
+=====================
+
+0.3.0 (February 2016)
+---------------------
+
+ * drops support for OMERO 5.1 and earlier
+ * remove `--ports` support. Use 5.2 properties.
```

Comment:
Since these are not full sentences, our style guide defines that all points should start with a lowercase letter and there should be no periods at the end. This specific point is difficult, maybe change to "remove… and use 5.2 properties" to avoid the issue?